aiida/transports/plugins/ssh.py

```diff
@@ -437,14 +437,26 @@ def open(self):
             )
             raise
 
-        # Open also a SFTPClient
-        self._sftp = self._client.open_sftp()
-        # Set the current directory to a explicit path, and not to None
-        self._sftp.chdir(self._sftp.normalize('.'))
+        # Open also a File transport client. SFTP by default, pure SSH in ssh_only
+        self.open_file_transport()
+
+        return self
+
+    def open_file_transport(self):
+        """
+        Open the SFTP channel, and handle error by directing customer to try another transport
+        """
+        from aiida.common.exceptions import InvalidOperation
+
+        try:
+            self._sftp = self._client.open_sftp()
+        except Exception:
+            raise InvalidOperation('SFTP seems not functional. Try using sshonly plugin instead?')
```

Comment:
Not sure if this message is super clear given that (1) people may not know that SFTP refers to what the default ssh plugin uses and (2) the process of actually using another plugin means you need to set up another copy of the computer with that different plugin. This is a cluster-configuration problem, right? Its not that the protocol might be temporary down. In that case what do you think of...?

```suggestion
            raise InvalidOperation('Error in ssh transport plugin. This may be due to the remote computer not supporting SFTP. Try setting it up with the sshonly plugin instead.')
```